python/python/tests/test_transaction.py

```diff
@@ -32,6 +33,7 @@ async def transaction_test(
     key7 = "{{{}}}:{}".format(keyslot, get_random_string(3))
     key8 = "{{{}}}:{}".format(keyslot, get_random_string(3))
     key9 = "{{{}}}:{}".format(keyslot, get_random_string(3))
+    key10 = "{{{}}}:{}".format(keyslot, get_random_string(3))
```

Comment:
maybe add a hint for that key
```suggestion
    key10 = "{{{}}}:{}".format(keyslot, get_random_string(3)) # list
```